api/auth.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const passport = require('../auth/index');
+const router = require('express').Router();
+
+router.get('/auth/facebook', passport.authenticate('facebook', { scope: 'email' }));
+router.get('/auth/facebook/callback', passport.authenticate('facebook',
+  { successRedirect: '/api/users', failureRedirect: '/login' }));
```

Comment:
Yes I wholeheartedly agree on this issue, the matter here is that I really didn't think this through enough but I knew I needed to implement this in a better way, but I just assumed that I will take care of it later, when I have more knowledge of the matter, or even when the frontend implementation was more clear.